QLogger.h

```diff
@@ -22,9 +22,9 @@
  ** Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  ***************************************************************************************/
 
-#include <QLoggerLevel.h>
+#include "QLoggerLevel.h"
```

Comment:
This needs to be reverted. APIs should always use angle brackets.

Reply:
No, this does not work with CMake, at least not in the kdenlive settings, so I need this like this.

Reply:
Yes, five or six arrows -> need to be replaced for a dot (.) I already mentioned that myself.

Reply:
@OndrejPopp what you have then it's a problem of CMake configuration, not C++. This should work as intended. In your CMakes, you'll need to att something like this:

`target_include_directories(target PRIVATE dir/to/QLogger)`.

The angle brackets have a specific use in C++ and are different to the quotes despite in this case the result might seem the same.

Change it, please.